src/codegen/mod.rs

```diff
@@ -1342,3 +1344,28 @@ impl From<&ast::Builtin> for Builtin {
         }
     }
 }
+
+pub(super) fn print_runtime_error(ns: &Namespace, error: &String, loc: Option<Loc>) -> String {
```

Comment:
Nit: better to use  a `&str` ref instead, otherwise I don't think this will work with static str refs for example.